Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -1235,14 +1235,14 @@ def lhegensim(fragment,howMuch):
 
 # step4
 step4Defaults = { 
-                  '-s'            : 'ALCA:TkAlMuonIsolated+TkAlMinBias+MuAlOverlaps+EcalESAlign',
+                  '-s'            : 'ALCA:TkAlMuonIsolated+TkAlMinBias+MuAlOverlaps+EcalESAlign+EcalTrg',
                   '-n'            : 1000,
```

Comment:
@shervin86 sorry. 
I didn't notice you added the new AlCaReco in MC as well. Will need to update the MC GT as well.